scripts/local-k8s/setup-ci.sh

```diff
@@ -22,6 +22,7 @@ export CODALAB_WORKER_MANAGER_CPU_BUNDLE_RUNTIME=kubernetes
 export CODALAB_WORKER_MANAGER_CPU_KUBERNETES_CLUSTER_HOST=https://codalab-control-plane:6443
 export CODALAB_WORKER_MANAGER_TYPE=kubernetes
 export CODALAB_WORKER_MANAGER_CPU_KUBERNETES_CERT_PATH=/dev/null
+export CODALAB_WORKER_MANAGER_CPU_KUBERNETES_CERT=/dev/null
```

Comment:
Why use /dev/null here (and check for that in the code) rather than just not set the environment variable and then check in the code if it's empty?

Reply:
Due to a quirk in how we handle environment variables, we aren't able to set them to empty values. This is because code such as here (https://github.com/codalab/codalab-worksheets/blob/master/docker_config/compose_files/docker-compose.yml#L303) will just print out `--cert-path --cpus ...`, which will cause a parsing error. Ideally, we should find a way to fix this later.